src/trace_processor/perfetto_sql/stdlib/android/frames/timeline.sql

```diff
@@ -274,7 +274,7 @@ WHERE
 -- information across different layers for a given frame_id in a given process.
 CREATE PERFETTO TABLE android_frames (
   -- Frame id.
```

Comment:
Can we add a note here explaining what the "frame id" is: where does it come from (it's a Android level concept which ends up in trace, not something we create on our side).